doc/contributing/collaborator-guide.md

```diff
@@ -392,13 +392,21 @@ metadata. Raise a pull request like any other change.
 
 Treat commits that introduce new core modules with extra care.
 
-Check if the module's name conflicts with an existing ecosystem module. If it
-does, choose a different name unless the module owner has agreed in writing to
-transfer it.
+New modules must only be added with the `node:` prefix.
```

Comment:
It seems very unfortunate to reify this when it seems a clear decision was not, in fact, made. Why can't this go to a new vote where the intention is made clear from the start?